### PR TITLE
Archived petitions improvement

### PR DESCRIPTION
**Pull request by mybits**
Detailed ACs in here:
PT: https://www.pivotaltracker.com/n/projects/307301/stories/97375798

### CODE REVIEW

**Comment by h-lame** [comment]
Tiny comment on the number of petitions, but this looks good (code - not checked what it looks like in browser) to me!

File: app/views/archived/petitions/index.html.erb [hunk 1]
```diff
@@ -1,16 +1,26 @@
...
-<div class="cta_block" id="create_pet_cta_block">
-  <h2>What are petitions?</h2>
-  <p>Petitions are an easy, personal way for you to influence government and Parliament in the UK. You can create a petition about anything that the government is responsible for and if it gets at least 100,000 signatures, it will be considered for debate in the House of Commons.  You can find more information about how the House of Commons deals with petitions on the <%= new_window_link_to "Backbench Business Committee website", "http://www.parliament.uk/bbcom" %>.</p>
-  <%= link_to 'How petitions work', help_path, :class => 'large_link' %>
-  <%= link_to "Create a new petition", check_petitions_path, :class => 'link_button large_link_button' %>
-</div>
+  <% if @petitions.empty? %>
+      <p class="no_petitions_warning">No archived petitions could be found.</p>
+  <% else %>
+    <form method="get" action="/archived/petitions/search" class="search-form search-inline">
```
Comment (otlaitil) [inline app/views/archived/petitions/index.html.erb]: This form could be rewritten with rails' `form_helper` (you can copy paste the one from my branch if you feel like it!)

**Comment by h-lame** [comment]
Let's rebase and merge this.

**Comment by mybits** [comment]
Hi @otlaitil and @h-lame - there was some changes in ACs for that story, but now it's ready for re-review :)

**Pull request merged**